commons/all/src/main/java/org/infinispan/commons/util/Util.java

```diff
@@ -680,17 +677,6 @@ public static int getNormalizedHash(Object object, Hash hashFct) {
       return hashFct.hash(object) & Integer.MAX_VALUE;
    }
 
-   /**
-    * Returns the size of each segment, given a number of segments. This assumes
-    * a 32 bit hash is used and we ignore the most significant bit.
-    *
-    * @param numSegments number of segments required
-    * @return the size of each segment
-    */
-   public static int getSegmentSize(int numSegments) {
```

Comment:
I worry that removal of these methods is technically part of our "public" API. I think we can only deprecate for now.

Reply:
TBH, I don't think these constitute "API", although we have not made it clear they are NOT for public consumption.

Reply:
They have a package info in there that says it is public :( I am fine with removing though as I doubt people are using these.